controllers/cluster_controller.go

```diff
@@ -543,7 +543,7 @@ func (r *FoundationDBClusterReconciler) updateOrApply(ctx context.Context, clust
 			return err
 		}
 
-		return r.Status().Patch(ctx, unstructuredPatch, client.Apply, client.FieldOwner("fdb-operator")) //, client.ForceOwnership)
+		return r.Status().Patch(ctx, unstructuredPatch, client.Apply, client.FieldOwner("fdb-operator"), client.ForceOwnership)
```

Comment:
The last time I looked at this the subresource didn't support the force ownership option, so it seems like this was fixed/changed in one of the controller-runtime versions.